### PR TITLE
a11y: Lightbox buttons accessibility

### PR DESCRIPTION
**Pull request by ramiy**
initial patch to fix accessibility issues on Elementor Lightbox.

## PR Checklist

- [x] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Lightbox buttons accessibility

## Description
An explanation of what is done in this PR

* Adds labels to Lightbox buttons.

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Comment by ramiy** [comment]
This PR adds `title` attribute to the `<i>` tag for screen readers.

```
<i class="icon-class" title="accessibility text"></i>
```

For now, the `title` attribute is enough.

----

For better accessibility, the buttons must follow the following pattern:

```
<div>
  <i class="icon-class" aria-hidden="true" title="accessibility text"></i>
  <span class="elementor-screen-only">accessibility text</span>
<div>
```

Elementor uses this pattern in most in it's code. But for now the title attribute is enough.

**Comment by ramiy** [comment]
Related: #2909, #2861 and all the [a11y PRs](https://github.com/elementor/elementor/pulls?q=is%3Apr++a11y+).

File: assets/dev/js/frontend/utils/lightbox/lightbox.js [hunk 1]
```diff
@@ -225,7 +225,7 @@ module.exports = elementorModules.ViewModule.extend( {
 		if ( ! videoUrl ) {
 			const downloadImage = i18n.downloadImage;
 
-			$linkList.append( $( '<a>', { href: itemUrl, download: '' } ).text( downloadImage ).prepend( $( '<i>', { class: 'eicon-download-bold' } ) ) );
+			$linkList.append( $( '<a>', { href: itemUrl, download: '' } ).text( downloadImage ).prepend( $( '<i>', { class: 'eicon-download-bold', title: 'Download' } ) ) );
```
Comment (matipojo) [inline assets/dev/js/frontend/utils/lightbox/lightbox.js:228]: The `title` attribute is visible on hover, and needs translation, please use `aria-label` here and all next.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/frontend/utils/lightbox/lightbox.js (hunk 1 above)
Comment (ramiy) [reply assets/dev/js/frontend/utils/lightbox/lightbox.js]: Done.

**Review by matipojo — DISMISSED** [review]

**Review by matipojo — DISMISSED** [review]

**Comment by ramiy** [comment]
@shilo-ey Can you merge?

**Comment by joshmarom** [comment]
Hi @ramiy. Please close this PR and open one to the `develop` branch.
Cheers

**Comment by ramiy** [comment]
This PR will be merged as part of #11283

**Pull request closed without merging**